client/src/components/accountInfo/accountInfo.jsx

```diff
@@ -1,20 +1,27 @@
-import React, { useState } from 'react';
-import { Button, TextField, Typography } from '@mui/material';
+import React, { useState, useEffect } from 'react';
+import { Button, TextField, Typography, Tooltip } from '@mui/material';
 import { Grid, Box } from '@mui/material';
 import { Link } from 'react-router-dom';
 import Avatar from '@mui/material/Avatar';
 import ProfilePic from '../../../dist/mockProfile.png';
+import axios from 'axios';
 
-function AccountInfo() {
+function AccountInfo(props) {
 
-   let mockDatabaseAccountNumber = 1234;
+   const [edit, setEdit] = useState(false);
 
-   let userInfo = {
-      firstName: "Daniel",
-      lastName: "Halper",
-      userName: "Dhalper",
-      email: "Dhalper@test.org",
-   }
+   let remainingFunds = 400;
+
+   const [userInfo, setUserInfo] = useState({
+      firstName: props.userInfo.firstName,
+      lastName: props.userInfo.lastName,
+      email: props.userInfo.email,
+      userName: props.userInfo.userName,
+      profilePic: props.userInfo.profilePic,
+      bank: props.userInfo.bank,
+      accountNumber: props.userInfo.accountNumber
+    });
+  
```

Comment:
Also, generally take a moment to think through what your use case is for putting state from a higher component also into state here. The same info is now in state for two different components?